File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/NotFoundException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;
import lombok.Getter;

@Getter
public class NotFoundException extends BusinessException {

    private String value;

    public NotFoundException(String value) {
        this(value, ErrorCode.NOT_FOUND);
        this.value = value;
    }

    public NotFoundException(String value, ErrorCode errorCode) {
        super(value, errorCode);
        this.value = value;
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/NotFoundException.kt
package com.dogGetDrunk.meetjyou.common.exception.business

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

open class NotFoundException(
    val value: String,
    errorCode: ErrorCode = ErrorCode.NOT_FOUND
) : BusinessException(value, errorCode)
